hathor/transaction/util.py

```diff
@@ -112,8 +116,11 @@ def validate_token_name_and_symbol(settings: HathorSettings, token_name: str, to
     if symbol_len == 0 or symbol_len > settings.MAX_LENGTH_TOKEN_SYMBOL:
         raise TransactionDataError('Invalid token symbol length ({})'.format(symbol_len))
 
-    # Can't create token with hathor name or symbol
+    # Can't create token with hathor name or symbol and version None
     if clean_token_string(token_name) == clean_token_string(settings.HATHOR_TOKEN_NAME):
         raise TransactionDataError('Invalid token name ({})'.format(token_name))
     if clean_token_string(token_symbol) == clean_token_string(settings.HATHOR_TOKEN_SYMBOL):
         raise TransactionDataError('Invalid token symbol ({})'.format(token_symbol))
+    # Only HTR can have version=None; all other tokens must define and validate their version.
+    if token_version is None:
```

Comment:
The method signature assumes that it will never be None. You can assert that it is not none to prevent misbehavior. If you agree and create a `NATIVE_TOKEN` type, you might assert that it's never a native token type.